### PR TITLE
chore: Typo cleanup

### PR DESCRIPTION
**Pull request by nschonni**
Last pass with cSpell. Think the dictionary is about ready to add after this

### CODE REVIEW

File: docs/integrate/api/_scripts/1.0/policyConfigurations.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ exports.submitRequests = function()
 
 	apiwriter.postJson('/build/definitions?api-version=2.0-preview.2',
 		{
-			// this is a rather incomplete build defininition, but we only need a valid ID, not a valid definition
+			// this is a rather incomplete build definitition, but we only need a valid ID, not a valid definition
```
Comment (steved0x) [inline docs/integrate/api/_scripts/1.0/policyConfigurations.js]: @nschonni Please check this one.

File: docs/integrate/previous-apis/_scripts/1.0/policyConfigurations.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ exports.submitRequests = function()
 
 	apiwriter.postJson('/build/definitions?api-version=2.0-preview.2',
 		{
-			// this is a rather incomplete build defininition, but we only need a valid ID, not a valid definition
+			// this is a rather incomplete build definitition, but we only need a valid ID, not a valid definition
```
Comment (steved0x) [inline docs/integrate/previous-apis/_scripts/1.0/policyConfigurations.js]: @nschonni definition

**Comment by steved0x** [comment]
@nschonni I left two review comments for `definitition`. I would fix them but it sounds like a spelling dictionary entry might need to be updated so I wanted to let you know. I'll merge it once those are fixed. Thank you for your contributions!

**Comment by nschonni** [comment]
I'll give it a check when I get home. Feel free to push the fix if you want to land it earlier.

**Comment by nschonni** [comment]
Fixed, didn't find it in a local dictionary so I'm not sure where it is coming in from

**Pull request merged**